changelogs/fragments/77020-galaxy-output-for-requirements-yml.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Ansible Galaxy - Collections and Roles can now be listed in a format compatible for the requirements.yml file
```

Using `ansible-galaxy` would make it more obvious that this is a change to the cli, not Galaxy itself.
```suggestion
  - ansible-galaxy - Collections and Roles can now be listed in a format compatible for the requirements.yml file
```

Roles didn't support any formatting before, so another item should be added to the list to mention that listing roles now support json and yaml formatting.